README.md

```diff
@@ -7,11 +7,21 @@
 ### 1. Clone this repository by running:
   * ```git clone https://github.com/td-org-uit-no/tdctl-api.git```
 
-### 2. Launching locally
+### 2. Developing
+You can either start up both the backend and mongodb locally, or you can run them in separate docker containers using docker-compose.
+
+**NB: Doing both may result in issues since docker and mongod will in some cases use different users for writing to the volume folder `db_data`. If this happen, you should either try to fix the permissions of the folder, or you can delete and recreate it**
```

Comment:
I think we could add a recommendation to choose one of the methods and stick to it

Reply:
Better now?

Reply:
yes, go a head and merge :+1: